inventory/technical/reporting.rst

```diff
@@ -0,0 +1,191 @@
+
+inventory valuation of current inventory will redirect you to
+http://amazing_stock.odoo.com/web?#action=912&active_id=1&model=product.product&view_type=list&menu_id=
+
+inventory valuation of current inventory will redirect you to
+http://amazing_stock.odoo.com/web?#active_id=2&model=product.product&view_type=list&menu_id=
+
+Both are in fact different views on the model product.product.
+
+Let's set the context
+***********************
+The main differences are in the context:
+
+The context is a strange odoo concept: we use for communication between the client and the server.
+
+You can 'easily' see the context sent to the server.
+When you look at the https(s) requests and inspect the headers:
+(in chrome, press F12 then open the tab network and look for the search_read)
+
+see the images for more details in the referenceA_ :
+
+context of inventory valuation of current inventory:
+
+
+.. code-block:: json
+
+    {
+    "jsonrpc":"2.0",
+    "method":"call",
+    "params":{
+        "model":"product.product",
+        "domain":[
+            ["type","=","product"],["qty_available","!=",0]
+        ],
+        "fields":[
+            "display_name",
+            "qty_at_date",
+            "uom_id",
+            "currency_id",
+            "cost_currency_id",
+            "stock_value",
+            "cost_method"
+        ],
+        "limit":80,
+        "sort":"",
+        "context":{
+            "lang":"en_US",
+            "tz":"Europe/Brussels",
+            "uid":2,
+            "valuation":true,
+            "active_model":"stock.quantity.history",
+            "active_id":7,
+            "active_ids":[7],
+            "company_owned":true,
+            "create":false,
+            "edit":false,
+            "search_disable_custom_filters":true
+        }
+    },
+    "id":431504480
+    }
+
+
+context of inventory valuation of current inventory:
+
+.. code-block:: javascript
+
+    {
+    "jsonrpc":"2.0",
+    "method":"call",
+    "params":{
+        "model":"product.product",
+        "domain":[
+            ["type","=","product"],["qty_available","!=",0]
+        ],
+        "fields":[
+            "display_name",
+            "qty_at_date",
+            "uom_id",
+            "currency_id",
+            "cost_currency_id",
+            "stock_value",
+            "cost_method"
+        ],
+        "limit":80,
+        "sort":"",
+        "context":{
+            "lang":"en_US",
+            "tz":"Europe/Brussels",
+            "uid":2,
+            "valuation":true,
+            "active_model":"stock.quantity.history",
+            "active_id":8,
+            "active_ids":[8],
+            "default_compute_at_date":0,
+            "to_date":"2019-08-09 10:59:29",
+            "company_owned":true,
+            "create":false,
+            "edit":false,
+            "search_disable_custom_filters":true
+        }
+    },
+    "id":925403830
+    }
```

Comment:
Can you explain which keys do matter for inventory valuation?

Reply:
done